docs/guides/upgrading-to-indie-linter-v2.md

```diff
@@ -0,0 +1,77 @@
+# Upgrading to Indie Linter v2
+
+This upgrade guide assumes your package uses Indie Linter API v1 and Message API v1.
```

Comment:
"package uses" -> "package currently uses"

And possibly change uses to implements, but then you would need to split out the mention of Message API so it's probably fine.